frontend/express/public/core/user-management/stylesheets/_drawer.scss

```diff
@@ -0,0 +1,266 @@
+.user-management-drawer-content {
```

Comment:
Thanks for this one.

Reply:
its missing cly-vue- in the beginning though.
all blocks should prepend **cly-vue-** in the beginning.
Its ofcourse only to avoid class naming conflicts with the current code.

Reply:
I think cly-vue rule has not been followed strictly. But I think it is not needed as long as the namespace makes it unique.

Reply:
yep